### PR TITLE
pcsx2-gui: Further improve with adding punctation,tooltip and formatting

### PR DESCRIPTION
**Pull request by RedDevilus**

### CODE REVIEW

File: pcsx2/gui/MainFrame.cpp [hunk 1]
```diff
@@ -383,15 +383,15 @@ void MainEmuFrame::CreatePcsx2Menu()
...
+	m_menuSys.Append(MenuId_Sys_SaveStates,	_("&Save State"), &m_SaveStatesSubmenu);
 
-	m_menuSys.Append(MenuId_EnableBackupStates,	_("&Backup before save"), wxEmptyString, wxITEM_CHECK);
+	m_menuSys.Append(MenuId_EnableBackupStates,	_("&Backup Before Save"), wxEmptyString, wxITEM_CHECK);
 
 	m_menuSys.AppendSeparator();
 
 	m_menuSys.Append(MenuId_Exit,			_("E&xit"),
-		AddAppName(_("Closing %s may be hazardous to your health")));
+		AddAppName(_("This will close %s.")));
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:394]: Don't change this one.

Comment (lightningterror) [reply pcsx2/gui/MainFrame.cpp]: You can still add the period at the end.

**Comment by lightningterror** [comment]
While you're at it you can also update this
https://github.com/PCSX2/pcsx2/blob/master/plugins/GSdx/Renderers/OpenGL/GLLoader.cpp#L401
Change `INFO` to `INFO:` for both lines.

File: pcsx2/gui/ConsoleLogger.cpp [hunk 2]
```diff
@@ -419,13 +419,13 @@ ConsoleLogFrame::ConsoleLogFrame( MainEmuFrame *parent, const wxString& title, A
 
 	// create Appearance menu and submenus
 
-	menuFontSizes.Append( MenuId_FontSize_Small,	_("&Small"),	_t("Fits a lot of log in a microcosmically small area."),
+	menuFontSizes.Append( MenuId_FontSize_Small,	_("&Small"),	_t("A smaller font size. (7px)"),
```
Comment (lightningterror) [inline pcsx2/gui/ConsoleLogger.cpp:422]: `7px font size.` instead?
Same for the rest.

File: pcsx2/gui/MainFrame.cpp [hunk 3]
```diff
@@ -352,29 +352,29 @@ void MainEmuFrame::CreatePcsx2Menu()
...
-		_("Skips PS2 splash screens when booting from ISO or DVD media"), wxITEM_CHECK );
+		_("Skips PS2 splash screens when booting from ISO or DVD media."), wxITEM_CHECK );
 
 	m_menuSys.AppendCheckItem(MenuId_Debug_CreateBlockdump, _("Create &Blockdump"), _("Creates a block dump for debugging purposes."));
 
 	m_menuSys.Append(MenuId_GameSettingsSubMenu,	_("&Game Settings"), &m_GameSettingsSubmenu);
 
 	m_GameSettingsSubmenu.Append(MenuId_EnablePatches,	_("Automatic &Gamefixes"),
-		_("Automatically applies needed Gamefixes to known problematic games"), wxITEM_CHECK);
+		_("Automatically applies needed Gamefixes to known problematic games (From gameindex.dbf)."), wxITEM_CHECK);
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:367]: Not needed, you can keep the period tho.

File: pcsx2/gui/MainFrame.cpp [hunk 4]
```diff
@@ -352,29 +352,29 @@ void MainEmuFrame::CreatePcsx2Menu()
...
 
 	m_GameSettingsSubmenu.Append(MenuId_EnableCheats,	_("Enable &Cheats"),
-		wxEmptyString, wxITEM_CHECK);
+		_("Use cheats otherwise known as pnachs (From cheats folder in Documents or PCSX2 folder itself)."), wxITEM_CHECK);
 
 	m_GameSettingsSubmenu.Append(MenuId_EnableWideScreenPatches,	_("Enable &Widescreen Patches"),
-		_("Enabling Widescreen Patches may occasionally cause issues."), wxITEM_CHECK);
+		_("Widescreen Patches may occasionally cause issues (From cheats_ws folder/zip)."), wxITEM_CHECK);
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:373]: Ditto.

File: pcsx2/gui/MainFrame.cpp [hunk 5]
```diff
@@ -352,29 +352,29 @@ void MainEmuFrame::CreatePcsx2Menu()
...
 
 #ifndef DISABLE_RECORDING
 	m_GameSettingsSubmenu.Append(MenuId_EnableInputRecording, _("Enable &Input Recording"),
-		wxEmptyString, wxITEM_CHECK);
+		_("Input Recording for controller/keyboard presses, tools for automation and play it back."), wxITEM_CHECK);
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:377]: And playback?

File: plugins/GSdx/Renderers/OpenGL/GLLoader.cpp [hunk 6]
```diff
@@ -325,7 +325,7 @@ namespace GLLoader {
 		if (!GLExtension::Has("GL_ARB_viewport_array")) {
 			glScissorIndexed   = ReplaceGL::ScissorIndexed;
 			glViewportIndexedf = ReplaceGL::ViewportIndexedf;
-			fprintf_once(stderr, "GL_ARB_viewport_array is not supported! Function pointer will be replaced\n");
+			fprintf_once(stderr, "GL_ARB_viewport_array is not supported! Function pointer will be replaced.\n");
```
Comment (lightningterror) [inline plugins/GSdx/Renderers/OpenGL/GLLoader.cpp:328]: Leave this one out.

**Comment by RedDevilus** [comment]
All changes done as requested.

File: 3rdparty/wxwidgets3.0/src/common/stockitem.cpp [hunk 7]
```diff
@@ -254,18 +254,18 @@ wxString wxGetStockHelpString(wxWindowID id, wxStockHelpStringClient client)
...
+        STOCKITEM(wxID_CUT,      wxSTOCK_MENU, _("Cut selection."))
+        STOCKITEM(wxID_DELETE,   wxSTOCK_MENU, _("Delete selection."))
+        STOCKITEM(wxID_REPLACE,  wxSTOCK_MENU, _("Replace selection."))
+        STOCKITEM(wxID_PASTE,    wxSTOCK_MENU, _("Paste selection."))
+        STOCKITEM(wxID_EXIT,     wxSTOCK_MENU, _("Quit this program."))
+        STOCKITEM(wxID_REDO,     wxSTOCK_MENU, _("Redo last action."))
+        STOCKITEM(wxID_UNDO,     wxSTOCK_MENU, _("Undo last action."))
+        STOCKITEM(wxID_CLOSE,    wxSTOCK_MENU, _("Close current document."))
+        STOCKITEM(wxID_SAVE,     wxSTOCK_MENU, _("Save current document."))
+        STOCKITEM(wxID_SAVEAS,   wxSTOCK_MENU, _("Save current document with a different filename."))
```
Comment (lightningterror) [inline 3rdparty/wxwidgets3.0/src/common/stockitem.cpp:268]: Don't change these.

File: pcsx2/gui/MainFrame.cpp [hunk 8]
```diff
@@ -450,7 +450,8 @@ void MainEmuFrame::CreateWindowsMenu()
 void MainEmuFrame::CreateCaptureMenu()
 {
 	m_menuCapture.Append(MenuId_Capture_Video, _("Video"), &m_submenuVideoCapture);
-	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Screenrecorder"));
+	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Screenrecorder"),	
+		_("Creates a video-only file and an audio-only wav file."));
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:454]: Not really needed.

Comment (RedDevilus) [reply pcsx2/gui/MainFrame.cpp:454]: Well the tooltip could lose partly it's meaning when GSdx/SPU2-x get merged.

**Comment by lightningterror** [comment]
Not sure what to do with this one, is it really worth breaking a bunch of translations just to add periods?

**Comment by RedDevilus** [comment]
There are already a bunch of translations broken, unless we go to a cloud solution or an update to pot file. I thought 1.6 was the go-to for people wanting translations along with the rest.

**Review by lightningterror — APPROVED** [review]

File: pcsx2/gui/MainFrame.cpp [hunk 9]
```diff
@@ -379,20 +379,20 @@ void MainEmuFrame::CreatePcsx2Menu()
...
 								 wxEmptyString, wxITEM_CHECK);
 
 	m_GameSettingsSubmenu.Append(MenuId_EnableWideScreenPatches, _("Enable &Widescreen Patches"),
-								 _("Enabling Widescreen Patches may occasionally cause issues."), wxITEM_CHECK);
+		_("Enabling Widescreen Patches may occasionally cause issues."), wxITEM_CHECK);
 
 #ifndef DISABLE_RECORDING
 	m_GameSettingsSubmenu.Append(MenuId_EnableInputRecording, _("Enable &Input Recording"),
-								 wxEmptyString, wxITEM_CHECK);
+		_("Input Recording for controller/keyboard presses, tools for automation and playback."), wxITEM_CHECK);
```
Comment (lightningterror) [inline pcsx2/gui/MainFrame.cpp:395]: Doesn't clang add the spaces?

Comment (RedDevilus) [reply pcsx2/gui/MainFrame.cpp:395]: Ive just looked at the other macros, and i remember them being like this.

Comment (lightningterror) [reply pcsx2/gui/MainFrame.cpp:395]: Let clang deal with the formatting.

**Comment by RedDevilus** [comment]
I did clang format, hope it's correct.

**Comment by lightningterror** [comment]
Can you properly name the formatting commit?

**Comment by lightningterror** [comment]
Maybe split the gsdx clang in it's own commit?

**Comment by lightningterror** [comment]
Forgot to mention what files are formatted with clang.

**Pull request merged**